x-pack/plugins/security_solution/public/flyout/shared/hooks/use_fetch_prevalence.ts

```diff
@@ -99,7 +102,10 @@ export const useFetchPrevalence = ({
     },
   } = useKibana();
 
-  const searchRequest = buildSearchRequest(highlightedFieldsFilters, from, to);
+  // retrieves '.alerts-security.alerts-default' and 'logs-*' indices
```

Comment:
i suggest saying `detection` and `non-detection` indices here (or general terms). we are not hardcoding indices